micro_dl/inference/evaluation_metrics.py

```diff
@@ -4,8 +4,13 @@
 import pandas as pd
 from skimage.metrics import structural_similarity as ssim
 import sklearn.metrics
+from skimage.morphology import disk, dilation, erosion
+from skimage.measure import label, regionprops
 from scipy.stats import pearsonr
 
+from cellpose import models, utils, io
+from lapsolver import solve_dense
+
```

Comment:
@Soorya19Pradeep Can you provide versions that you depend on for these libraries? I know that sklearn and cellpose both have had some breaking API changes over the recent years.

Reply:
Installed the versions @Soorya19Pradeep provided, excpet for lapsolver, for which I installed `1.1.0` instead of `1.1.0=dev_0` since the later seems to be a typo.